dev/integration_tests/ui/integration_test/integration_test.dart

```diff
@@ -2,16 +2,17 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:integration_ui/build_mode.dart' as app;
+import 'package:integration_ui/overflow.dart' as app;
 
 void main() {
   group('Integration Test', () {
     testWidgets('smoke test', (WidgetTester tester) async {
         app.main();
         await tester.pumpAndSettle();
 
-        expect(find.text('Hello, world!'), findsOneWidget);
+        expect(find.byType(SizedBox), findsOneWidget);
```

Comment:
I picked another entry point for this smoke test so `build_mode` could be deleted.